Cabal/Distribution/Simple/GHCJS.hs

```diff
@@ -72,6 +72,9 @@ import Distribution.Types.ForeignLib
 import Distribution.Types.ForeignLibType
 import Distribution.Types.ForeignLibOption
 import Distribution.Types.UnqualComponentName
+#if MIN_VERSION_base(4,9,0)
```

Comment:
why this is `#if`d?

Reply:
To make it work on 7.x versions of the compiler, for unclear reasons.  My guess now is that Edmund initially started importing Data.List.NonEmpty per file and then later added the re-export in Cabal's Prelude, so these are redundant.